loc/items/gem.js

```diff
@@ -2,76 +2,77 @@ export default [
   {
     name       : {en: 'Lucid' },
     description: {
-      en: 'Gem description'
+      en: 'The sharp green edges catch sparkling light in the sun, making a beautiful and deadly sight.'
```

Comment:
For the first 4-5 gems i would try to go along with what we were doing with the  other items. make the descriptions less "amazing", for this one i would change "beautiful and deadly" into something more humorous.